solutions.md

```diff
@@ -60,9 +60,29 @@ clean :
 > 3. only .dat files recreated
 > 4. only analysis.tar.gz recreated
 
+~~~ 
```

Comment:
Oh I'll fix that! Sorry. Should I simply remove these markers or add a different class?